overrides/config/bosshealthregain-common.toml

```diff
@@ -2,7 +2,7 @@
 #General Settings
 [Bosses]
 	#List of entities that should regain health whenever a play dies nearby.
-	"boss entities:" = ["minecraft:wither", "minecraft:ender_dragon", "minecraft:elder_guardian", "meetyourfight:swampjaw", "meetyourfight:bellringer", "meetyourfight:dame_fortuna", "meetyourfight:rosalyne", "predators:scarab", "predators:prisnautilus", "predators:huntsman", "predators:rock_boss", "iter_rpg:insatiable", "iter_rpg:sorrowsealed", "graveyard:lich", "queen_bee:queen_bee", "born_in_chaos_v1:sir_pumpkinhead", "alexsmobs:warped_mosco", "cataclysm:netherite_monstrosity", "cataclysm:the_leviathan", "cataclysm:ignis", "cataclysm:the_harbinger", "cataclysm:ender_golem", "cataclysm:ender_guardian", "alexmobs:void_worm", "witherstormmod:wither_storm", "unusualend:endstone_golem", "aquamirae:captain_cornelia", "blue_skies:alchemist", "blue_skies:arachnarch", "blue_skies:starlit_crusher", "blue_skies:summoner", "conjurer_illager:conjurer", "mutantmonsters:mutant_creeper", "mutantmonsters:mutant_enderman", "mutantmonsters:mutant_skeleton", "mutantmonsters:mutant_zombie", "mutantmore:mutant_blaze", "mutantmore:mutant_shulker", "mutantmore:mutant_wither_skeleton"]
+	"boss entities:" = ["minecraft:wither", "minecraft:ender_dragon", "minecraft:elder_guardian", "meetyourfight:swampjaw", "meetyourfight:bellringer", "meetyourfight:dame_fortuna", "meetyourfight:rosalyne", "predators:scarab", "predators:prisnautilus", "predators:huntsman", "predators:rock_boss", "iter_rpg:insatiable", "iter_rpg:sorrowsealed", "graveyard:lich", "queen_bee:queen_bee", "born_in_chaos_v1:sir_pumpkinhead", "alexsmobs:warped_mosco", "cataclysm:netherite_monstrosity", "cataclysm:the_leviathan", "cataclysm:ignis", "cataclysm:the_harbinger", "cataclysm:ender_golem", "cataclysm:ender_guardian", "alexmobs:void_worm", "witherstormmod:wither_storm", "unusualend:endstone_golem", "aquamirae:captain_cornelia", "blue_skies:alchemist", "blue_skies:arachnarch", "blue_skies:starlit_crusher", "blue_skies:summoner", "conjurer_illager:conjurer", "mutantmonsters:mutant_creeper", "mutantmonsters:mutant_enderman", "mutantmonsters:mutant_skeleton", "mutantmonsters:mutant_zombie", "mutantmore:mutant_blaze", "mutantmore:mutant_shulker", "cataclysm:ancient_remnant", "bosses_of_mass_destruction:lich", "bosses_of_mass_destruction:gauntlet", "bosses_of_mass_destruction:void_blossom", "bosses_of_mass_destruction:obsidilith", "mutantmore:mutant_wither_skeleton"]
 	#Radius to check around player who died for bosses to regain health. Default 16. Value between 0 and 256
```

Comment:
not sure why github doesnt highlight it properly but the firrence between these two arrays is 

"cataclysm:ancient_remnant", "bosses_of_mass_destruction:lich", "bosses_of_mass_destruction:gauntlet", "bosses_of_mass_destruction:void_blossom", "bosses_of_mass_destruction:obsidilith",